docs/installer/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
if we land this i would love to do something where it's not just an installer page but actually just the landing page for wasmpack, e.g. links to the docs, shows an example... no need to actually change this but thought i'd mention

Reply:
👍